### PR TITLE
Metis support for the Layer2 Sequencer Health EA

### PR DESCRIPTION
**Pull request by cds95**
## Description

Adds support to fetch the health of the Metis Sequencer using the Layer2 Sequencer Health EA

## Changes

- Add `metis` as an option for `network` when calling the Metis Sequencer EA

## Steps to Test

1. Run EA
2. Call EA and verify that it returns the latest Sequencer health

## Quality Assurance

- [x] Ran `yarn changeset` if adapter source code was changed
- [x] If a new adapter was made, or an existing one was modified so that its environment variables have changed, update the relevant `<ADAPTER_PACKAGE>/schemas/env.json` and `<ADAPTER_PACKAGE>/README.md`
- [x] If a new adapter was made, or an existing one was modified so that its environment variables have changed, update the relevant `infra-k8s` configuration file.
- [x] The branch naming follows git flow (`feature/x`, `chore/x`, `release/x`, `hotfix/x`, `fix/x`) or is created from Clubhouse/Shortcut
- [x] This is related to a maximum of one Clubhouse/Shortcut story or GitHub issue
- [x] Types are safe (avoid TypeScript/TSLint features like any and disable, instead use more specific types)

### CODE REVIEW

File: packages/sources/layer2-sequencer-health/README.md [hunk 1]
```diff
@@ -39,13 +41,17 @@ For the adapter to be useful on the desired network, at least one endpoint (RPC
...
+  "result": 0,
+  "statusCode": 200,
   "data": {
-    "result": true,
-    "isHealthy": true
-  },
-  "statusCode": 200
+    "isHealthy": 0,
+    "result": 0
+  }
```
Comment (boxhock) [inline packages/sources/layer2-sequencer-health/README.md:55]: This doesn't sit right with me:
`"isHealthy": 1 == sequencer is unhealthy`

Can we improve this? Change to `isUnhealthy`?

**Review by boxhock — CHANGES_REQUESTED** [review]

File: packages/sources/layer2-sequencer-health/README.md (hunk 1 above)
Comment (cds95) [reply packages/sources/layer2-sequencer-health/README.md:55]: What about if we just return the Boolean value for isHealthy? I don’t think we can switch the 0/1 value for the result field as we already have feeds that are dependent on 0 representing a healthy sequencer and 1 representing an unhealthy sequencer.

Comment (boxhock) [reply packages/sources/layer2-sequencer-health/README.md:55]: Yeah that could work as well. I was not suggesting that we change the 0/1 value, but rather rename the return param `isHealthy` to `isUnhealthy`. I'm guessing we're only using the `result` param, so renaming this additional param wouldn't have any effect?

Comment (cds95) [reply packages/sources/layer2-sequencer-health/README.md:55]: Gotcha. Yeah think that we’re only using the result param so this should not have any effect.

**Review by boxhock — APPROVED** [review]

**Pull request merged**